iowrappers/redis_client.go

```diff
@@ -627,7 +627,7 @@ func (r *RedisClient) SavePlanningSolutions(ctx context.Context, request *Planni
 			Logger.Error(err)
 			continue
 		}
-		_, recordSaveErr := r.client.Set(ctx, solutionRedisKey, json_, 0).Result()
+		_, recordSaveErr := r.client.Set(ctx, solutionRedisKey, json_, PlanningSolutionsExpirationTime).Result()
```

Comment:
This was defined on the previous PR I believe

Reply:
No, the constant `PlanningSolutionsExpirationTime` existed before if this is what you mean.